public-api/src/main/resources/public-api.yaml

```diff
@@ -82,7 +104,57 @@ paths:
           description: A collection of domains
           schema:
             $ref: "#/definitions/DbDomainListResponse"
+  /v1/databrowser/domain-totals:
+    get:
+      security: []
+      tags:
+        - dataBrowser
+      description: Gets the domain filters with the count of matched concepts
+      operationId: "getDomainTotals"
+      parameters: []
+      responses:
+        200:
+          description: A collection of domains
+          schema:
+            $ref: "#/definitions/DbDomainListResponse"
 
+  /v1/databrowser/advanced-concept-search:
+    get:
+      security: []
+      tags:
+        - dataBrowser
+      description: Gets list of matched concepts
+      operationId: "getAdvancedConceptSearch"
+      parameters:
+        - in: query
+          name: keyword
+          type: string
+          required: true
+          description: search word
+        - in: query
+          name: conceptFilter
+          type: string
+          required: false
+          description: concept filter to filter standard concepts
+        - in: query
+          name: vocabularyIds
+          type: array
+          items:
+            type: string
+          required: false
+          description: vocabulary ids
+        - in: query
+          name: domainIds
+          type: array
+          items:
+            type: string
```

Comment:
Similarly, you may want to switch to use the Domain enum I defined in workbench at some point. (I don't think researchers should have to specify the underlying domain IDs, since that's exposing some messiness in OMOP.)

Reply:
If you do decide to do this, we should probably move this mapping into ConceptService:

https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/api/ConceptsController.java#L28

and move Domain and StandardConceptFilter enums into some shared YAML file that we can merge into both APIs.